onboard/import.go

```diff
@@ -26,12 +29,13 @@ type Importer struct {
 type Config struct {
 	CommitUsername     string
 	InventoryURL       string
-	Repository         string
+	RepoID             graveler.RepositoryID
+	Repo               *catalog.Repository
```

Comment:
Don't think this one should be part of the configuration if we already passing the RepositoryID.
We need the full Repository information because we do not have access to the entry catalog - ask just Reposoty without passing the repository ID.

Reply:
Passed just what's mandatory - default branch id